### ISSUE TITLE
Confusing error messages on Windows

### ISSUE THREAD

**Issue by rpavlik** [vrpn/vrpn#38]
In a number of places, it appears that error messages say "no error" instead of their actual error message. I suspect this may be because of the macro-driven errno substitution, but I have not messed with it to check.

One place is when accessing a missing serial port.

Another is opening a second VRPN server on the same computer:

```
open_socket:  can't bind address 3883  --  0  --  No error
  (This probably means that another application has the port open already)
open_socket:  can't bind address 3883  --  0  --  No error
  (This probably means that another application has the port open already)
```

**Comment by russell-taylor** [comment]
Windows sockets don't set errno on errors, they require a special call, WSAGetLastError().  errno still works for file operations, but they have a separate group for networking.  How exciting.

**Comment by russell-taylor** [comment]
Fixed in commit 15d195c0f4c5eaba4a3a5dc5d47ff68526b879b1, which has been pushed to master.
